python/scrimmage/bindings/src/py_openai_env.cpp

```diff
@@ -355,8 +358,34 @@ void ScrimmageOpenAIEnv::seed(pybind11::object _seed) {
     seed_ = _seed.cast<int>();
 }
 
-pybind11::tuple ScrimmageOpenAIEnv::step(pybind11::object action) {
+void ScrimmageOpenAIEnv::filter_ext_ctrl_vec(){
```

Comment:
can we just use ``erase`` with ``remove_if``? This would look something like:

```[c++]
auto no_parent = [&](auto &p){return p->parent() == nullptr;};
std::erase(std::remove_if(actions_.ext_ctrl_vec().begin(), actions_.ext_ctrl_vec().end(), no_parent), actions_.ext_ctrl_vec().end());
```